.github/workflows/backend.yml

```diff
@@ -59,13 +59,16 @@ jobs:
           DB_NAME: ztmf
           DB_USER: admin
           DB_PASS: localDev 
+          DB_POPULATE: "/src/backend/_test_data.sql"
```

Comment:
Is this of any risk to show

Reply:
nope its dummy test data that does not contain any production data